hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -1101,6 +1105,10 @@ struct shlq { Vreg64 s, d; VregSF sf; Vflags fl; }; // uses rcx
 /*
  * arm intrinsics.
  */
+struct csoneb { ConditionCode cc; VregSF sf; Vreg8 t, d; };
```

Comment:
I thought we couldn't have PhysRegs in the vasm-instrs.  

The peephole flips the condition code if necessary and only creates the
pseudo if semantically correct.